### ISSUE TITLE
[bug] Can no longer edit Text value

### ISSUE THREAD

**Issue by akivajgordon** [tldraw/tldraw#972]
First, what a fantastic and beautiful app. I love how simple it is.

TL;DR – I can not edit text from an existing text element

(reproduced on Safari 15.5 and Chrome 103)

## Reproduction Steps

1. Add "Text" element to the canvas
2. Type any text value, say "foobar"
3. Unfocus the Text element
4. Double-click to try to edit

## Expected Behavior

After double-clicking, the Text element becomes editable and I can start typing to overwrite the text

## Actual Behavior

Keys I press behave as if the Text item is not selected. As an example, if I try to type "status", the "Sticky" element gets selected after pressing "s", then the "Text" element gets selected after pressing "t", and so on.

I discovered this after starting a new tldraw document, but since I actually had an open tab of tldraw for something I was working on yesterday, I attempted to edit text in that tab and confirmed that the behavior was ok. But then once I refreshed the page, I could no longer edit the text. So I assume this bug was introduced very recently.

**Comment by vastchris** [comment]
Yes,I also met this problem on Edge.Hope to be able to fix it as soon as possible

**Comment by akivajgordon** [comment]
Just an update here, a `git bisect` tells me that the issue was introduced in commit 7774d00e745294d9ea583b1364955fedaf9d6604. I'm investigating what that might have been.

**Comment by judicaelandria** [comment]
Hey @akivajgordon do you want to work on this one and send a pr? :)

**Comment by judicaelandria** [comment]
nm it's already fixed!
